docs/api-routes/response-helpers.md

```diff
@@ -98,6 +98,8 @@ export default function handler(
 }
 ```
 
+> Note: You cannot type the body of `NextApiRequest` by design. You should run the input through a schema validator to generate a typed object for consumption.
```

Comment:
```suggestion
> Note: The body of `NextApiRequest` is any because the client may include any payload. You should validate the type/shape of the body at runtime before using it. 
```